# Data acquisition time and frame id
std_msgs/Header header

# cluster CAN messages
conti_ars408_side2/radar_cluster_info[] cluster_info